PusherClient/EventEmitter.cs

```diff
@@ -28,6 +28,26 @@ public void BindAll(Action<string, dynamic> listener)
             _generalListeners.Add(listener);
         }
 
+        public void Unbind(string eventName)
+        {
+            _eventListeners.Remove(eventName);
+        }
+
+        public void Unbind(string eventName, Action<dynamic> listener)
+        {
+            if(!_eventListeners.ContainsKey(eventName))

```

Comment:
I suppose this could be simplified with 

```
if(_eventListeners.ContainsKey(eventName))
   _eventListeners[eventName].Remove(listener);
}
```